docs/src/main/paradox/api-reference/acls.md

```diff
@@ -0,0 +1,138 @@
+# Access control lists
+
+Access to any resource in the system is protected by ACLs.  Assuming a nexus deployment at
+`http(s)://nexus.example.com` and resource address of `/v0/{address}` the following operations should apply
+to all resources for authenticated users:
+
+## Authentication and authorization
+
+Access to the ACL on a particular resource is protected by the ACL itself.
+
+This implies that the caller (i.e. the client sending an HTTP request to perform any of the following actions)
+must have the `own` permission for administrative tasks (*create*, *add*, *subtract*, *clear*, *fetch all*) and
+the `read` permission to fetch its own permissions.
+
+See @ref:[Authentication](auth.md) to learn about the authentication mechanism.
+
+## Bootstrapping
+
+By default, the IAM service gives `read` and `own` rights to the LDAP groups defined as administrators in the
+system internal configuration settings.
+
+This is done by setting permissions to the top-level, root resource path `/` for these groups.  The root ACL
+can be subsequently modified by users belonging to administrator groups.
+
+**Note**: Be careful when modifying top-level permissions, as it can results in locking everyone out, or
+on the contrary, giving full ownership to everyone, including anonymous users.  No checks about the sanity of
+such operations are performed by the service.
+
+## Actions
+
+### Fetch own permissions on a resource
+
+```
+GET /v0/acls/{address}
+```
+
+**Note**: Permissions are transitively inherited from the resource ACL and all its ancestors.
+This call computes the effective ACL for the caller on this resource.
+
+#### Status Codes
+
+- **200 OK**: the ACL is computed and the associated caller permissions are returned successfully
+- **403 Forbidden**: the caller doesn't have `read` rights on this resource
+
+## Administrative actions
+
+### Fetch permissions for all users on a resource
+
+```
+GET /v0/acls/{address}?all=true
+```
+
+**Note**: This action *does not* show inherited permissions.
+
+#### Status Codes
+
+- **200 OK**: the entire ACL is fetched is returned successfully
+- **403 Forbidden**: the caller doesn't have `own` rights on this resource
+
+### Create a new ACL
+
+```
+PUT /v0/acls/{address}
+{...}
+```
+
+Payload
+:   @@snip [acls.json](../assets/api-reference/acls.json)
```

Comment:
It would be interesting on any resource to have 3 tabs (Request | Payload | Response).

Whenever there is not Payload, then just Request and Response should be provided.